pkg/helm/notify/notify.go

```diff
@@ -0,0 +1,183 @@
+/*
+Copyright The Helm maintainers
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package notify // import "k8s.io/helm/pkg/helm/notify"
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/Masterminds/semver"
+	"k8s.io/helm/pkg/version"
+)
+
+var (
```

Comment:
do we wanna make this a const instead if it's a private variable? I don't see anywhere in the package where we are going to change the time layout.

Reply:
Now a bad idea. Will do that in next update.